packages/lesswrong/components/comments/CommentOnPostWithReplies.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useCallback, useState } from 'react';
+import { useRecordPostView } from "../common/withRecordPostView";
+import { CommentWithRepliesProps } from "./CommentWithReplies";
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+
+interface CommentOnPostWithRepliesProps extends CommentWithRepliesProps {
+  post: PostsBase;
+}
+
+const CommentOnPostWithReplies = (props: CommentOnPostWithRepliesProps) => {
+  const post = props.post;
```

Comment:
```suggestion
const CommentOnPostWithReplies = ({post, ...otherProps}: CommentOnPostWithRepliesProps) => {
```

[nit] To me this makes it more clear what you're going to do